integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForUpdateCommand.scala

```diff
@@ -52,6 +53,13 @@ private[sql] case class CarbonProjectForUpdateCommand(
       return Seq.empty
     }
     val carbonTable = CarbonEnv.getCarbonTable(databaseNameOp, tableName)(sparkSession)
+    val isLoadInProgress = SegmentStatusManager.checkIfAnyLoadInProgressForTable(carbonTable)
+    if (isLoadInProgress) {
+      LOGGER.error("Cannot run data loading and update on same table concurrently. Please wait"
+                   + " for load to finish")
+      throw new Exception("Cannot run data loading and update on same table concurrently. " +
```

Comment:
Use the `ConcurrentOperationException` added in #1711

Reply:
@jackylk I will update this code once #1711 will be merged.

Reply:
#1711 is merged